draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2242,13 +2286,15 @@ In addition to the above parameters, the mapping f may require
 additional parameters Z, M, rational\_map, E', and/or iso\_map.
 These MUST be specified when applicable.
 
-Suites with nonuniform encodings MUST NOT be used in applications whose security
-relies on a random oracle. Applications using nonuniform encodings SHOULD carefully
-analyze the security implications of using such an encoding. When the required
-encoding is not clear, applications SHOULD use a random oracle.
+The below table lists suites RECOMMENDED for some elliptic curves.
+The corresponding parameters are given in the following subsections.
+Suites that use nonuniform encodings MUST NOT be used in protocols whose security
```

Comment:
We flip from using applications before the suite details, to protocols after. I don't understand the difference. (A protocol whose security depends on hash-to-curve is an application of hash-to-curve, in my view.)

Reply:
I revisited this with fresh eyes and pushed some more edits.